src/typeahead/typeahead.spec.ts

```diff
@@ -441,7 +384,7 @@ class TestComponent {
   private _objects =
       [{id: 1, value: 'one'}, {id: 10, value: 'one more'}, {id: 2, value: 'two'}, {id: 3, value: 'three'}];
 
-  model = '';
+  model: any = '';
```

Comment:
One of the tests has

    comp.model = {};

That does not compile anymore if the model is a string.